src/main/kotlin/com/pubnub/api/endpoints/objects/member/ManageChannelMembers.kt

```diff
@@ -10,7 +10,6 @@ import com.pubnub.api.models.consumer.objects.member.PNMemberArrayResult
 import com.pubnub.api.models.server.objects_api.ChangeMemberInput
 import com.pubnub.api.models.server.objects_api.EntityArrayEnvelope
 import com.pubnub.api.models.server.objects_api.MemberInput
-import com.pubnub.api.models.server.objects_api.UUIDId
 import com.pubnub.extension.toPNMemberArrayResult
 import retrofit2.Call
 import retrofit2.Response
```

Comment:
Why we have this line:
import com.pubnub.api.models.consumer.objects.member.MemberInput as ConsumerMemberInput
>

Reply:
To distinguish two different MemberInput types. One is in `consumer` package and the other is in `server` package

Reply:
I renamed one of them, so it's not required to use alias anymore